src/main/java/io/stargate/sgv2/jsonapi/service/schema/tables/CQLSAIIndex.java

```diff
@@ -4,6 +4,7 @@
 import com.datastax.oss.driver.api.core.metadata.schema.IndexKind;
 import com.datastax.oss.driver.api.core.metadata.schema.IndexMetadata;
 import com.datastax.oss.driver.internal.core.adminrequest.AdminRow;
+import com.datastax.oss.driver.internal.core.util.Strings;
```

Comment:
It seems dangerous to use classes from driver's "internal" packages....

Reply:
We use it in other places though.
What do you recommend Tatu, writing our own unquote() method is better?

Reply:
No, let's use that method for now.